src/pages/oauth-view/oauth-view.js

```diff
@@ -1,46 +1,53 @@
-import OAuthLoginZone from "../../components/oauth-login"
-import Auth0ClientProvider from '../../auth/auth0-client-provider.js'
-import { useEffect, useState, useContext } from "react"
-import axios from 'axios'
-
+import { useEffect, useState } from 'react'
+import { Redirect } from 'react-router'
+import './oauth-view.scss'
+import routes from '../../routes'
+import qs from 'qs'
+import LoadingElement from '../../components/generic-elements/loading-element'
+import PropTypes from 'prop-types'
+
+const hasCodeParameter = search => {
```

Comment:
It's unclear to me what the "code" is

Reply:
This is an authorization code.  We will rename it to reflect this.